internal/app/machined/pkg/controllers/cluster/local_affiliate.go

```diff
@@ -185,10 +185,15 @@ func (ctrl *LocalAffiliateController) Run(ctx context.Context, r controller.Runt
 						spec.KubeSpan.AdditionalAddresses = append([]netaddr.IPPrefix(nil), ksAdditionalAddresses.(*network.NodeAddress).TypedSpec().Addresses...)
 
 						nodeIPs := addresses.(*network.NodeAddress).TypedSpec().IPs()
-						endpoints := make([]netaddr.IPPort, len(nodeIPs))
+						endpoints := make([]netaddr.IPPort, 0, len(nodeIPs))
```

Comment:
I wish there were a way to lint these kinds of errors.  This seems to get me _every_ time.

Reply:
I wouldn't ever even _use_ preallocation if it weren't for that _other_ linter which requires it.  The Go memory allocator is, IMO, good enough.